tests/unit/coordinator_instance.cpp

```diff
@@ -52,11 +49,11 @@ class CoordinatorInstanceTest : public ::testing::Test {
 //
 //   auto const coord_to_replica_config =
```

Comment:
@antoniofilipovic already fixed this but I made a mistake with rebase, he will revert when comes back